src/test/java/picard/sam/RevertSamTest.java

```diff
@@ -445,4 +443,14 @@ public void testGetDefaultExtension() {
         Assert.assertEquals(RevertSam.getDefaultExtension("this.is.a.bam"), ".bam");
         Assert.assertEquals(RevertSam.getDefaultExtension("foo"), ".bam");
     }
+
+    @Test(expectedExceptions = PicardException.class)
+    public void testNoRgInfo() {
```

Comment:
Actually, `UNPAIRED_FASTQ` is meant to output reads that are unpaired (FLAG != 0x1), not paired reads that are missing the mate So,  this needs to be tested with files containing either pairs of reads or unpaired reads that do not have the first FLAG bit set.